### ISSUE TITLE
Heads up on buttplug android

### ISSUE THREAD

**Issue by qdot** [dreadhonk/ToyControl#1]
Hi! Buttplug.io lead dev here.

You are probably already aware via the last commits and archived state of the buttplug-android repo, but that def won't be getting any updates. That said, I'm trying to figure out how to support Buttplug-rs on android via an FFI layer so we can keep it modern. If you happen to be really good at JNI and also like extreme amounts of pain due to JNI, please get in touch, because I have no damn clue what I am doing.

**Comment by dreadhonk** [comment]
@qdot yep, I'm aware. thanks for the hint, but I have absolutely no clue about JNI :).

**Comment by qdot** [comment]
We had someone working on android bluetooth support for blteplug and buttplug, so we're... somewhat closer, but they've disappeared at the moment. Check the repos at https://github.com/gedgygedgy/
